server/datastore/mysql/hosts_test.go

```diff
@@ -210,12 +210,13 @@ func TestSaveHostPackStats(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, host.PackStats, 2)
 
+	// FIXME: since we don't remove all stats when we insert new ones, the check below is not true anymore
 	// Set to empty should make it empty
-	host.PackStats = []fleet.PackStats{}
-	require.NoError(t, ds.SaveHost(host))
-	host, err = ds.Host(host.ID)
-	require.NoError(t, err)
-	require.Len(t, host.PackStats, 0)
+	//host.PackStats = []fleet.PackStats{}
+	//require.NoError(t, ds.SaveHost(host))
+	//host, err = ds.Host(host.ID)
+	//require.NoError(t, err)
+	//require.Len(t, host.PackStats, 0)
```

Comment:
I'm not sure this needs to be a FIXME -- Any stats for a scheduled query on a host should be deleted by the `ON DELETE CASCADE` when the scheduled query is deleted. Otherwise IMO it's fine for the stats to stick around (indicating that the query remains scheduled but for some reason we did not receive new stats for it).